.infrastructure/clusterIP.yml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: todoapp-service
+  namespace: todoapp
+spec:
+  selector:
+    app: todoadd
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
+  type: ClusterIP
```

Better to specify `type` of service after `spec`
```suggestion
spec:
  type: ClusterIP
  selector:
    app: todoadd
  ports:
    - protocol: TCP
      port: 80
      targetPort: 8080
  type: ClusterIP
```